Makefile

```diff
@@ -79,53 +75,50 @@ compose-frontend:
 container-frontend-push: container-frontend
 	docker push ${CONTAINER_FRONTEND_TAG}
 
-# WARNING: EXPERIMENTAL feature to merge frontend and backend in one container
+## Backend + Frontend container
 container-tornjak: bin/tornjak-backend #ui-agent
 	docker build --no-cache -f Dockerfile.tornjak-container --build-arg version=$(VERSION) -t ${CONTAINER_TAG} .
 
 container-tornjak-push: container-tornjak
 	docker push ${CONTAINER_TAG}
 
 
-# releases for Github Container Registry
-release-tornjak-backend-ghcr: container-tornjak-backend
-	docker tag ${CONTAINER_BACKEND_TAG} ${CONTAINER_BACKEND_GHCR_IMAGEPATH}:latest
-	docker tag ${CONTAINER_BACKEND_TAG} ${CONTAINER_BACKEND_GHCR_IMAGEPATH}:$(VERSION)
+## BEGIN RELEASES FOR GITHUB CONTAINER REGISTRY ##
+# These targets are used by Github to create official pre-built images
+
+## backend image
+release-tornjak-backend: container-tornjak-backend
```

Comment:
So here, maybe we should rename it to
`container-backend-push-test` ? And then we tag and push accordingly?

Reply:
sticking with the `release` prefix because it corresponds to Github releases